Doc/library/tkinter.rst

```diff
@@ -788,7 +788,8 @@ Bitmap/Pixelmap images can be created through the subclasses of
 
 * :class:`BitmapImage` can be used for X11 bitmap data.
 
-* :class:`PhotoImage` can be used for GIF and PPM/PGM color bitmaps.
+* :class:`PhotoImage` can be used for GIF and PPM/PGM bitmaps. This class can
+  also be used for PNG bitmaps starting with Tcl/Tk 8.6.
 
```

Comment:
I don't know why 'bitmap' is used for image files.  Compressed files are not bitmaps as I understand the term.  So I think this should say (without the markup)

class PhotoImage can be used with GIF and PPM/PGM files.  PNG files can also be used, starting with Tcl/Tk 8.6.

Or replace 'file' with 'format' or 'data'.

Reply:
LGTM.  Lets see when Serhiy thinks.

Reply:
I would write something like: "... for images in PPM/PGM, GIF and PNG formats.  The latter is only supported in Tk 8.6."

Reply:
I agree. I'll update the PR.

Reply:
Just a small change: "... for images in PPM/PGM, GIF and PNG formats. The latter is supported starting with Tk 8.6."

To be correct regarding the upcoming 8.7.

ok?